proc_Hahn_echoph.py

```diff
@@ -55,10 +48,7 @@
     fl.image(s)
     #}}}
     #{{{select coherence and select t2 axis range
-    s = s['ph1',1]['ph2',0]
-    s.ift('t2')
-    s = s['t2':(0,None)]
-    s.ft('t2')
+    s = select_FID_and_slice...
     #}}}
```

Comment:
seems to overlap with select FID and slice

Reply:
replaced with appropriate function slice_FID_from_echo and runs fine. Only issue is it plots the same thing several times but I can not figure out why (pun intended)

Reply:
in vim, you can do `/\(plot(\|\<next(\|\<image(\)` as well as `\<fl\>` to scan through the plotting commands